pinot-common/src/main/java/org/apache/pinot/common/function/FunctionConstants.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.common.function;
+
+public class FunctionConstants {
+  private FunctionConstants() {
+  }
+
+  /**
+   * Maximum of json path cache size.
+   * the number of JSON paths is bounded by the size of the transformation config,
+   * add this threshold for protection from the extreme cases.
+   */
+  public static final int MAX_JSON_PATH_CACHE_SIZE = 1024 * 16;
```

Comment:
This class should be removed and the constant inlined into `JsonPathMapCache` where it is relevant.